documentation/modules/running-migration-plan.adoc

```diff
@@ -14,19 +14,21 @@ You can run a migration plan and view its progress in the {ocp} web console.
 
 .Procedure
 
-. Click *Migration plans*.
+. In the {ocp} web console, click *Migration* -> *Plans for virtualization*.
 +
-The *Migration plans* list displays the source and target providers, the number of virtual machines (VMs) being migrated, and the status of the plan.
+The *Plans* list displays the source and target providers, the number of virtual machines (VMs) being migrated, the status, and the description of each plan.
 
 . Click *Start* beside a migration plan to start the migration.
+. Click *Start* in the confirmation window that opens.
++
+The *Migration details by VM* screen opens, displaying the migration's progress
 +
 Warm migration only:
 
 * The precopy stage starts.
 * Click *Cutover* to complete the migration.
 
-. Expand a migration plan to view the migration details.
+. If the migration fails, click *Get logs* to view the migration logs, then click *Get logs* in the confirmation window that opens.
+. Click a migration's *Status*, whether it failed or succeeded, to view the details of the migration.
 +
-The migration details screen displays the migration start and end time, the amount of data copied, and a progress pipeline for each VM being migrated.
-
-. Expand a VM to view the migration steps, elapsed time of each step, and its state.
```

Comment:
I think this line should be kept. The steps, elapsed time and state are not shown by default, needs to expand the VM to see them.

Reply:
@qiyuann I rephrased line 32 -- the new text is line 34.

Reply:
I saw line 34, but it's missing the expanding content.

Reply:
My mistake -- I looked only at my failed documentation migration. When I tried a successful migration, I saw the difference.